src/Sulu/Bundle/WebsiteBundle/Resolver/ParameterResolver.php

```diff
@@ -114,6 +117,7 @@ public function resolve(
             $localizations[$locale] = [
                 'locale' => $locale,
                 'url' => $url,
+                'country' => $localization->getCountry(),
```

Comment:
just tested this - of we add `country`.  i think it would be nice to add `'language' => $localization->getLanguage()` here too. would you mind doing that in this PR? 🙂